### DISCUSSION TITLE
Xcode Cloud with .xcconfig(ft. 삽질⛏️)

### DISCUSSION THREAD

**Discussion by daminoworld** [DeveloperAcademy-POSTECH/.github#67] [Idea]
3기 비전입니다
제가 예전에 XcodeCloud 사용하면서 씨름했던 내용입니다ㅎㅎ

https://medium.com/daily-monster/xcode-cloud-with-xcconfig-ft-%EC%82%BD%EC%A7%88-%EF%B8%8F-4d1aae000604

**Comment by yeniful** [comment]
공유해주셔서 감사합니다!!
